serial_driver/include/serial_driver/serial_bridge_node.hpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 LeoDrive, Copyright 2021 the Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SERIAL_DRIVER__SERIAL_BRIDGE_NODE_HPP_
+#define SERIAL_DRIVER__SERIAL_BRIDGE_NODE_HPP_
+
+#include "serial_driver/serial_driver.hpp"
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_lifecycle/lifecycle_node.hpp>
+#include <lifecycle_msgs/msg/state.hpp>
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "msg_converters/converters.hpp"
+
+namespace lc = rclcpp_lifecycle;
+using LNI = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface;
+using example_interfaces::msg::UInt8MultiArray;
```

Comment:
Shouldn't this use `serial_msgs`? https://github.com/flynneva/serial_msgs

Reply:
I looked around for something similar but `UInt8MultiArray` was the closest I could find. We can always switch to it later. I expect that most people won't use the bridge node anyway and will, instead, use the `SerialDriver` directly. Leaving for now.